communication/slack-config/channels.yaml

```diff
@@ -241,6 +241,7 @@ channels:
   - name: sig-openstack
   - name: sig-pm
   - name: sig-release
+  - name: sig-release-branch-mgmt
```

Comment:
```suggestion
  - name: release-managers
```

Reply:
With that name, would it be used for more than just branch management?

It might be a tight fit, but if that is the name to be used, I'd still opt for the `sig-` prefix to fall inline with our general naming conventions (e.g. all the `sig-docs-*` channels)

Reply:
It would be Patch Release Managers + Branch Managers + SIG Chairs + maybe PSC.

If there is an accepted nomenclature, we should document that. I quickly looked, so I could be missing it, but I didn't see any.
Also, SIG Docs, just for the sake of example, has several channels that don't conform: https://github.com/kubernetes/community/blob/e92605e50d41fa5cc1c4f04ac4de8db09d9df92b/communication/slack-config/sig-docs/docs-channels.yaml#L1-L11

Would definitely prefer `release-managers` over `sig-release-release-managers`. :stuck_out_tongue_closed_eyes:

Reply:
Derp, yeah. They use those for all the docs channels. I was thinking of sig-docs-tools etc 🤦‍♂ I guess if anything going forward would be `release-*`

Reply:
@mrbobbytables -- Hehe. I can group our channels in a follow-up PR, so ownership is clear.

Reply:
Awesome. 👍 sgtm.